test/integration/scheduler/framework_test.go

```diff
@@ -892,10 +892,13 @@ func TestBindPlugin(t *testing.T) {
 		},
 	}
 
-	// Create the master and the scheduler with the test plugin set.
+	// Create the scheduler with the test plugin set.
 	testCtx := testutils.InitTestSchedulerWithOptions(t, testContext, false, nil, time.Second,
 		scheduler.WithProfiles(prof),
 		scheduler.WithFrameworkOutOfTreeRegistry(registry))
+	testCtx.InformerFactory.Start(testCtx.Ctx.Done())
```

Comment:
perhaps put this in `testCtx.SyncInformerFactory`

Reply:
Do you mean
```go
func (textCtx *TestContext) SyncInformerFactory() {
	testCtx.InformerFactory.Start(testCtx.Ctx.Done())
	testCtx.InformerFactory.WaitForCacheSync(testCtx.Ctx.Done())
}
```
?
I agree it's quite repetitive and duplicative to repeat those two lines.

Reply:
Yes

Reply:
What about
```go
// SyncInformerFactory starts informer and waits for caches to be synced
func SyncInformerFactory(testCtx *TestContext) {
	testCtx.InformerFactory.Start(testCtx.Ctx.Done())
	testCtx.InformerFactory.WaitForCacheSync(testCtx.Ctx.Done())
}
```
instead. I don't see any method defined for TestContext (or under test/integration/util/util.go). Though, I can see `func CleanupTest(t *testing.T, testCtx *TestContext)` which takes `TestContext` as an argument and alike.

Reply:
either is fine.